### PR TITLE
Add visually hidden page headers to summary templates

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Adds hard coded visually hidden table headers to the `ListCollector`, `Hub` and `Summary`/`SectionSummary` templates in order to help screen readers navigate the columns.

List collector: `Name of person`, `Action`

List summary (within question): `Name of person`

Hub: `Name of section or person`, `Section progress`, `Access section`

Section/Final summary: `Question`, `Answer given`, `Change answer`

### How to review 
Check that the new hidden headers have been added and match the Trello Card.

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[Add Visually hidden Table headers for all tables (S)](https://trello.com/c/0DWF5yeI/4305-add-visually-hidden-table-headers-for-all-tables-s)

File: tests/integration/questionnaire/test_questionnaire_list_collector_primary_person.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 class TestQuestionnaireListCollector(IntegrationTestCase):
     def get_link(self, row_index, text):
-        selector = f"tbody:nth-child({row_index}) td:last-child a"
+        selector = f"tbody:nth-child({row_index + 1}) td:last-child a"
```
Comment (ajmaddaford) [inline tests/integration/questionnaire/test_questionnaire_list_collector_primary_person.py:10]: Using nth child wasn't ideal previously, but adding one to skip the header is even less clear.

There is a common `get_link` method in `QuestionnaireTestCase`, which should be used. To do this, you'll need to:
- change the base class of `IntegrationTestCase` to `QuestionnaireTestCase`
- change the call `self.get_link(1, "Change")` below to `self.get_link("change", 1)` 

It's also worth considering removing the `get_section_summary_link` method at the same time, as it looks like could also just use the `get_link` method.

File: templates/hub.html [hunk 2]
```diff
@@ -24,6 +24,7 @@ <h1 class="u-mt-m">{{ content.title }}</h1>
         {
           "groups": [
             {
+              "headers":["Name of section or person", "Section Progress", "Access Section"],
```
Comment (ajmaddaford) [inline templates/hub.html:27]: The capitalisation doesn't match the card:
```suggestion
              "headers":["Name of section or person", "Section progress", "Access section"],
```

File: templates/partials/summary/summary.html [hunk 3]
```diff
@@ -10,6 +10,7 @@
         "groups": [
           {
             "groupTitle": '<div id="' + group.id +'">' + group.title + '</div>' if group.title else None,
+            "headers": ["Question", "Answer Given", "Change Answer"],
```
Comment (ajmaddaford) [inline templates/partials/summary/summary.html:13]: The capitalisation doesn't match the card:
```suggestion
            "headers": ["Question", "Answer given", "Change answer"],
```

File: tests/integration/questionnaire/test_questionnaire_same_name_items.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 class TestQuestionnaireSameNameItems(IntegrationTestCase):
     def get_link(self, row_index, text):
-        selector = f"tbody:nth-child({row_index}) td:last-child a"
+        selector = f"tbody:nth-child({row_index + 1}) td:last-child a"
```
Comment (ajmaddaford) [inline tests/integration/questionnaire/test_questionnaire_same_name_items.py:8]: The same comment about the common `get_link` method applies here too.

File: tests/integration/questionnaire/test_questionnaire_list_collector_primary_person.py (hunk 1 above)
Comment (berroar) [reply tests/integration/questionnaire/test_questionnaire_list_collector_primary_person.py]: I replaced these methods, thanks!

**Review by ajmaddaford — APPROVED** [review]

**Review by petechd — APPROVED** [review]

**Pull request merged**